### PR TITLE
EVG-6445 Add commit message expansion to commit queue

### PR DESCRIPTION
**Pull request by bsamek**

### CODE REVIEW

File: model/project.go [hunk 1]
```diff
@@ -828,6 +828,18 @@ func PopulateExpansions(t *task.Task, h *host.Host, oauthToken string) (util.Exp
 		expansions.Put("revision_order_id", fmt.Sprintf("%s_%d", v.Author, v.RevisionOrderNumber))
 		if v.Requester == evergreen.MergeTestRequester {
 			expansions.Put("is_commit_queue", "true")
+			patch, err := patch.FindOne(patch.ByVersion(t.Version))
```
Comment (ablack12) [inline model/project.go]: Seems shady to use the library name as the variable

File: model/project.go [hunk 2]
```diff
@@ -828,6 +828,18 @@ func PopulateExpansions(t *task.Task, h *host.Host, oauthToken string) (util.Exp
...
+			if err != nil {
+				return nil, errors.Wrapf(err, "error finding patch for version %s", t.Version)
+			}
+			if patch == nil {
+				return nil, errors.Errorf("no patch found for version %s", t.Version)
+			}
+			for _, p := range patch.Patches {
```
Comment (john-m-liu) [inline model/project.go]: Would using patch.Description be the same thing?

File: model/project.go (hunk 1 above)
Comment (tychoish) [reply model/project.go]: +1

Comment (bsamek) [reply model/project.go]: Fixed.

File: model/project.go (hunk 2 above)
Comment (bsamek) [reply model/project.go]: Yup. Fixed.

File: model/project.go [hunk 3]
```diff
@@ -828,6 +828,18 @@ func PopulateExpansions(t *task.Task, h *host.Host, oauthToken string) (util.Exp
...
+				if p.ModuleName == "" {
+					expansions.Put("commit_message", p.Message)
```
[the remark this answers is not in the document]
Comment (bsamek) [reply model/project.go]: Used p.Description instead.

**Pull request merged**